configuration/configuration.go

```diff
@@ -57,6 +57,9 @@ type Configuration struct {
 			// contain the private portion for the file specified in
 			// Certificate.
 			Key string `yaml:"key,omitempty"`
+
+			// Specifies the CA cert for client authentication
+			CACert string `yaml:"ca,omitempty"`
```

Comment:
Do we know why they only accept a single certificate authority? It would be nice to expose the full extent of what is available in the tls package.